RELEASE_NOTES.md

```diff
@@ -1,5 +1,43 @@
 # Push Jobs Client Release Notes
 
+## 3.0
+
+Versions of Chef Push Jobs 3.0 and later may only be used under the terms of the [Chef EULA](https://www.chef.io/end-user-license-agreement/) or another commercial agreement with Chef Software.
+
+Dependency updates:
+- Ruby 2.6.6
+- Chef Infra Client 15.10.12
+- ohai 15.9.1
+- bzip2 1.0.8
+- cacerts 2019-10-16
+- libarchive 3.4.2
+- libxml2 2.9.10
+- libxslt 1.1.34
+- openssl 1.0.2u
+
+Added platform support:
+- AIX 7.2
+- Debian 9, 10
+- RHEL 8
+- macOS 15
+- Ubuntu 16.04, 18.04, 20.04
```

Comment:
Did we really only support 14.04 before?

Reply:
no. also 12.04 and 10.04 apparently. heh. we didn't have a `.expeditor/release.omnibus.yml` for the last release,